### ISSUE TITLE
Import via command line is not working

### ISSUE THREAD

**Issue by adius** [3dcitydb/3dcitydb#3]
I wrote an node.js wrapper for the command line interface to import citygml xml files into a postgres database. Everything seems to work, but the table doesn't contain any entries in the end.
When using the 3dcitydb-importer-exporter there is a popup during the import process which lists the building as they are imported. This step seems to missing in the command line version.
Do you have any ideas how to fix this?

```sh
$ cityviz import test/fixtures/cityGml/4-52-NOORD_KETHEL.xml
/System/Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home/bin/java -Xms128m -Xmx768m -jar /Users/adrian/Projects/CityViz/CityViz/node_modules/3dcitydb-importer-exporter/lib/3dcitydb-impexp.jar -shell -config /var/folders/bj/0twmh2ws4tn_t2sxn5vgqvmm0000gn/T/115223-5801-82sq2y -import /Users/adrian/Projects/CityViz/CityViz/test/fixtures/cityGml/4-52-NOORD_KETHEL.xml
[21:53:08 INFO] Starting 3D City Database Importer/Exporter, version "1.6-postgis-b1"
[21:53:08 INFO] Loading plugins
[21:53:08 INFO] Initializing application environment
[21:53:13 INFO] Loading project settings
[21:53:13 INFO] Connecting to database profile 'cityviz'.
[21:53:14 INFO] Database connection established.
[21:53:14 INFO] PostgreSQL, 9.4.0
[21:53:14 INFO] SRID: 28992 (Projected)
[21:53:14 INFO] SRS: Amersfoort / RD New
[21:53:14 INFO] gml:srsName: 28992
[21:53:14 INFO] Initializing database import...
[21:53:14 INFO] Spatial indexes are enabled.
[21:53:14 INFO] Normal indexes are enabled.
[21:53:14 INFO] Creating list of CityGML files to be imported...
[21:53:14 INFO] List of import files successfully created.
[21:53:14 INFO] 1 file(s) will be imported.
[21:53:15 INFO] Importing file: /Users/adrian/Projects/CityViz/CityViz/test/fixtures/cityGml/4-52-NOORD_KETHEL.xml
[21:53:15 INFO] Resolving XLink references.
[21:53:15 INFO] Cleaning temporary cache.
[21:53:15 INFO] Database import successfully finished.
```

**Comment by clausnagel** [comment]
I could not reproduce your problem on my machine. I get a summary of imported features and the tables are correctly filled.

```sh
c:\3dcitydb-impexp-1.6-postgis>java -jar lib\3dcitydb-impexp.jar -shell -import c:\test\4-52-NOORD_KETHEL.xml -config c:\test\config.xml
[11:18:01 INFO] Starting 3D City Database Importer/Exporter, version "1.6-postgis-b56"
[11:18:02 INFO] Loading plugins
[11:18:02 INFO] Initializing application environment
[11:18:09 INFO] Loading project settings
[11:18:10 INFO] Connecting to database profile 'test'
[11:18:11 INFO] Database connection established.
[11:18:11 INFO] PostgreSQL, 9.4.0
[11:18:11 INFO] SRID: 28992 (Projected)
[11:18:11 INFO] SRS: Amersfoort / RD New
[11:18:11 INFO] gml:srsName:
[11:18:11 INFO] Initializing database import...
[11:18:12 INFO] Spatial indexes are enabled.
[11:18:12 INFO] Normal indexes are enabled.
[11:18:12 INFO] Creating list of CityGML files to be imported...
[11:18:12 INFO] List of import files successfully created.
[11:18:12 INFO] 1 file(s) will be imported.
[11:18:14 INFO] Importing file: c:\Users\cnagel\Desktop
[11:18:17 INFO] Resolving XLink references.
[11:18:18 INFO] Resolving appearance XLinks...
[11:18:20 INFO] Importing texture images...
[11:20:21 INFO] Cleaning temporary cache.
[11:20:24 INFO] Imported CityGML features:
[11:20:24 INFO] APPEARANCE: 96
[11:20:24 INFO] BUILDING: 96
[11:20:24 INFO] GROUND_SURFACE: 96
[11:20:24 INFO] ROOF_SURFACE: 96
[11:20:24 INFO] WALL_SURFACE: 96
[11:20:24 INFO] Processed geometry objects: 3344
[11:20:24 INFO] Database import successfully finished.
```

Could you please attach your config file (make sure to delete database login credentials)?  I assume that you use the official Rotterdam [open data dataset] (http://www.rotterdamopendata.nl/dataset/rotterdam-3d-bestanden/resource/3c27fea1-7977-45a4-9e22-803da87b5217)? Or have you made any changes to this dataset?

Also note that the Rotterdam data uses texture atlases. The current version of the 3D City Database (2.1) and of the Importer/Exporter (1.6) do not support texture atlases. Thus, the texture image is imported anew for every geometry that references it. This may quickly exhaust your table space. The good news is that we have fixed this issue for the upcoming version 3.0. The release is still scheduled for end of this month!

**Comment by clausnagel** [comment]
Closing this issue due to lack of response. Please feel free to re-open if you want us to look into this.
